helm/charts/determined/templates/NOTES.txt

```diff
@@ -19,13 +19,15 @@ kubectl get service determined-master-service-{{ .Release.Name }} -n {{ .Release
 {{ end -}}
 
 {{ $httpOrHttps := "http" }}
```

Comment:
Per the thread, if they make no changes and don't read the docs, this silently fails to set their secret. Include an error message if they still have `.Values.tlsSecret` set.

Reply:
I was looking for an error function, but of course, Helm (and go templates) are never what anyone really wants. :D

My current best idea is to use [required](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function) in a random template file to depend on the new value if the old is set; maybe something along the lines of this untested blob of horror:
```plain
{{- if .Values.tlsSecret }} 
{{- required "tlsSecret has moved to security.tslSecret" .Values.security.tlsSecret }}
{{- end }}
```
I don't know for sure if that works in a conditional, though, so :shrug:

Reply:
Oh, sprig has a fail function.  So maybe `{{- fail "tlsSecret has moved to security.tslSecret" }}` would be a similar valid option to always fail if the old value is provided  I kinda like continuing to work if the new one is also added.  Dealer's choice, IMO.

Reply:
The decision was made to move `tlsSecret` back to it's original home and avoid the situation altogether.